extensions/compose/src/compose-extension.ts

```diff
@@ -135,6 +135,21 @@ export class ComposeExtension {
       this.statusBarItem.show();
       this.extensionContext.subscriptions.push(this.statusBarItem);
     }
+    const description = `Compose is a tool for defining and running multi-container Docker applications. 
```

Comment:
`compose-extension.ts` will be removed in the near future after we 100% get compose onboarding in, the reason being is that `compose-extension.ts` is mainly for the compose status bar on the bottom of Podman Desktop which will be removed in favourite for notifications.

Reply:
can you move it to `extension.ts`?

you can also re-use the same description for `extension.ts`

Reply:
you'll be able to use the same description once https://github.com/containers/podman-desktop/pull/4179 is added since it'll add the compose "provider" to the list.

Reply:
Moved to PR https://github.com/containers/podman-desktop/pull/4388